Packs/GitHub/Integrations/GitHub/GitHub.py

```diff
@@ -1834,6 +1834,56 @@ def fetch_incidents_command_rec(start_time, last_time, page=1):
     return incidents, last_time
 
 
+def get_path_data():
```

Comment:
Shouldn't be `get_file_data`?

Reply:
It works for path as well. Not only file so I included it

Reply:
See the README example!